src/test/java/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -245,10 +361,17 @@ public static void givePlayerRadar(final PlayerID player) {
     TechAttachment.get(player).setAARadar(Boolean.TRUE.toString());
   }
 
+  /**
+   * Helper method to check if a String is null and otherwise print the String.
+   */
   public static void assertValid(final String string) {
     Assert.assertNull(string, string);
   }
-
+  
+  /**
+   * Helper method to check if a String is null and otherwise print the String
+   * which doesn't make any sense because the String will then always be null.
+   */
   public static void assertError(final String string) {
```

Comment:
I think this method is asserting that an error is present, where the specified string is probably an error message or something.  That is, if no error occurred, the string would be `null`; if an error occurred, the string would be non-`null`.

Regardless, if you keep the existing Javadoc, the phrase "if a String is null" in the first sentence should probably be "if a String is **not** null".